arduinoTest/arduinoEcho/arduinoEcho.ino

```diff
@@ -2,7 +2,7 @@ const int ledPin = 11;
 
 void setup() {
   pinMode(ledPin, OUTPUT);
-  Serial.begin(9600);
+  Serial.begin(115200);
```

Comment:
Sorry, I think we are misunderstanding each other. Perhaps I can be clearer if I use full sentences. 

The PR I submitted has a .js file, SerialDuplexTest.js, and an Arduino sketch, ArduinoEcho.ino, both of which operate at 115200 bps.  That's all I have for this PR.

They're at 115200bps because I wanted to see how the test performed at what I usually run as top speed for the Arduino. That should be all you need.

In your comment, you mentioned that the default was 9600, and I agreed, and pointing out that  other examples I've written (for Arduino, not for node-serialport) operate at a default of 9600bps. There are no other examples I've written to accompany this particular PR, however. 

Are you asking for a complete list of examples I've written that use Serial.begin(9600)? If so, here you go: https://github.com/arduino/Arduino/search?utf8=%E2%9C%93&q=Tom+Igoe+9600

Or are you asking that I switch these particular examples back to 9600 from 115200?  Or do you want other examples to go with this one?  

I'm happy to comply with any of the above, but I'm not sure what you're asking for. I'm also happy to let this PR go, as I'm not especially attached to it. If it's not useful, feel free to reject it.